[Feat] 스페이스 회원은 투표의 후보를 추가할 수 있다.

Nine-JH opened the pull request:
## 변경사항
- 스페이스 회원들이 투표 후보를 추가하는 기능을 구현합니다.
- 단건 뿐 아니라 리스트로 추가가 가능합니다.

### Issue Link - #83

Nine-JH committed:
[Feat] 스페이스 회원은 투표의 후보를 추가할 수 있다.

motivation:
- 스페이스 회원들이 투표 후보를 추가하는 기능을 구현합니다.
- 단건 뿐 아니라 리스트로 추가가 가능합니다.

modification:
- 후보를 추가하는 `VoteManageUseCase.addCandidate` 유스케이스 추가
- 후보를 추가하는 API 구현

Close #83

Dr-KoKo reviewed (approved):
수고하셨습니다.

happymink reviewed (approved):
고생하셨습니다!

laigasus commented on app/src/main/java/fc/be/app/domain/vote/controller/dto/CandidateAddApiRequest.java:
```diff
@@ -0,0 +1,12 @@
+import static fc.be.app.domain.vote.service.VoteManageUseCase.CandidateAddRequest.CandidateInfo;
+
+public record CandidateAddApiRequest(
+        @NotNull List<CandidateInfo> datas
```
`datas` 다음 PR때 볼께요 ㅎㅎ

laigasus reviewed (approved):
수고하셨습니다! 🙇

Nine-JH committed:
[Refactor] request dto renaming

motivation:
- 추상적인 datas 보다는 명확한 이름으로 변경하는 것이 사용자에게 더욱 코드를 잘 이해할 수 있다고 판단해 이름을 변경합니다.

modification:
- `datas` -> `candidateInfos` 로 renaming

Close #83
